### ISSUE TITLE
IOS: Semantic issue

### ISSUE THREAD

**Issue by exmav10** [voximplant/react-native-voximplant#23]
I get this error 'Property has a previous declaration' for 20 times.

**Comment by ArtsNCode** [comment]
Hi @exmav10 ,
Did you solve the issue?

**Comment by grigoryvp** [comment]
@ArtsNCode If you have such an issue, please specify how to reproduce it so we can test it ourselves at Voximplant

**Comment by ArtsNCode** [comment]
Thank you @grigoryvp for your response.
I just created a fresh empty project and installed the latest version of this repo.

-After installation, I have followed the manual installation and everything gonna fine.
-After building the project I got the following two errors:

> Duplicate interface definition for class RCTView.
> Property has previous declaration

And i fixed it by replacing:#import "RCTView.h" with #import <React/RCTView.h> (VoxView.h) and #import "RCTBridge.h" with #import <React/RCTBridge.h> (VOXImplantModule.h).
-Then after building, I got the following error:

> ld: library not found for -lVoxImplantSDK
> clang: error: linker command failed with exit code 1 (use -v to see invocation)

Note that I am not using cocopods or any other dependency.

Xcode: 8.0
OSX el captain 10.11.6
react-native: 0.43.4

Thank you.
